how-thp-slows-down-your-database-performance-and-what-to-do-about-it.md

```diff
@@ -0,0 +1,115 @@
+---
+title: How THP Slows Down Your Database Performance and What to Do about It
+author: ['Wenbo Zhang']
+date: 2020-12-10
+summary: This post dives deep into how transparent huge pages (THP) slow down the system and explains why you should disable THP to improve your database performance.
+tags: ['Performance tuning', 'Linux']
+categories: ['Engineering']
+image: /images/blog/how-thp-slows-down-your-database-performance-banner.jpg
+---
+
+![Why We Disable THP](media/how-thp-slows-down-your-database-performance-banner.jpg)
```

Comment:
```suggestion
![Disabling THP to improve database performance](media/how-thp-slows-down-your-database-performance-banner.jpg)
```

Reply:
"improve database performance" may be an SEO keyword, so I suggest adding it here.